quick-start.md

```diff
@@ -0,0 +1,205 @@
+We will go through an example of **finding**, **accessing** and then **analyzing** some x-ray data data.
+
+We will be using 4 NuSTAR observation of **Cyg-X1**.
+
+<div style='color: #333; background: #ffffdf; padding:20px; border: 4px solid #fadbac'>
+<b>Running On Sciserver:</b><br>
+When running this notebook inside Sciserver, make sure the HEASARC data drive is mounted when initializing the Sciserver compute container. <a href='https://heasarc.gsfc.nasa.gov/docs/sciserver/'>See details here</a>.
+<br><br>
+<b>Running Outside Sciserver:</b><br>
+This notebook runs in the (heasoft) conda environment on Sciserver.
+If running outside Sciserver, some changes will be needed, including:<br>
+&bull; Make sure heasoftpy and heasoft are correctly installed (<a href='https://heasarc.gsfc.nasa.gov/docs/software/lheasoft/'>Download and Install heasoft</a>).<br>
+&bull; Unlike on Sciserver, where the data is available locally, you will need to download the data to your machine.<br>
+</div>
+
+
+## 2. Module Imports
+We need the following python modules:
+
+
+```python
+import glob
+
+# for finding data
+import pyvo
+
+# import nupipeline from heasoftpy
+# for heasoftpy version >= 1.4, it is under heasoftpy.nustar.nupipeline
+# for heasoftpy version < 1.4, it is under heasoftpy.nupipeline
+try:
+    from heasoftpy.nustar import nupipeline
+except ModuleNotFoundError:
+    from heasoftpy import nupipeline
+```
+
+## 3. Finding and Exploring the data
+
+The Heasarc data holdings can be searched and explored in different ways:
+- Using [Xamin Web Interface](https://heasarc.gsfc.nasa.gov/xamin/xamin.jsp).
+
+- Using a virtual observatory (VO) client such as [pyVO](https://github.com/astropy/pyvo) (see below) or [Topcat](http://www.star.bris.ac.uk/~mbt/topcat/).
+
+- Using the classical [Browse Mission Interface](https://heasarc.gsfc.nasa.gov/cgi-bin/W3Browse/w3browse.pl).
+
+In [Section 3.1](#3.1-pyvo-example) below, we give an example on how to use `pyVO` to search for NuSTAR data on a specific object. Alternatively, [Section 3.2](#3.2-using-xamin) assumes you can use Xamin to obtain a list of observations you are interested in. For more details on finding and accessing data, see the [notebook on finding and downloading data](data_find_download.md).
+
+The outcome of sections 3.1 and 3.2 is the same, so you can follow either of them.
+
+<!-- #region jp-MarkdownHeadingCollapsed=true -->
+### 3.1 Using Virtual Observatory Client pyVO:
+<!-- #endregion -->
+
+We first search the Virtual Observatory (VO) *registry* for data provided by `heasarc`. The registry provides an index of all data providers that allow access using VO standards.
+
+In the following example (`heasarc_service`), we search for all entries in the registry that have the keyword `heasarc`. This can a large and general set. The search can be filtered for more specfic datasets.
+
+```python
+import pyvo as vo
+
+heasarc_service = vo.regsearch(keywords='heasarc')
```

Comment:
For now, I'd change these to something like this:

```
all_services = vo.regsearch(servicetype='scs')
heasarc_services = []
for srv in all_services: 
    if 'heasarc' in srv.ivoid:
        heasarc_services.append(srv)
print(f'The search returned {len(heasarc_services)} HEASARC tables.')

master = []
for srv in heasarc_services: 
    if 'mast' in srv.ivoid:
        master.append(srv)
        print(f'{srv.ivoid}:\t {srv.res_title}')
print(f'The search returned {len(master)} "master" tables.')
```

Reply:
Fixed in 750b8c3b5fe94425589e854af779782f144ebfa8

Reply:
I went back to the original code since the issue is fixed at the backend.